CMakeLists.txt

```diff
@@ -330,7 +330,15 @@ target_link_libraries(${PROJECT_NAME} ${LINK_LIBRARY_LIST})
 set_target_properties(${PROJECT_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
 if(RPP_AUDIO_SUPPORT)
-    target_link_libraries(${PROJECT_NAME} ${PROJECT_SOURCE_DIR}/third_party/ffts/lib/libffts.a)
+    find_package(FFTS QUIET)
+    if(FFTS_FOUND)
+        target_link_libraries(${PROJECT_NAME} FFTS::FFTS)
+        set(RPP_AUDIO_AUGMENTATIONS_SUPPORT 1)
+        message("-- ${White}${PROJECT_NAME} -- Using FFTS - Library: ${FFTS_LIBRARIES} Include: ${FFTS_INCLUDE_DIRS}${ColourReset}")
+    else()
+        message("-- ${Yellow}RPP_AUDIO_SUPPORT set to build without FFTS${ColourReset}")
```

Comment:
Is FFTS a hard req for all audio augmentation. Then the message should say RPP is built without audio support

Reply:
done